src/main/resources/static/css/style.css

```diff
@@ -0,0 +1,12 @@
+.main-div {
+    padding-top:2%;
+    padding-bottom:2%;
+}
+
+.collection-div {
+    background-color:#0082C4;
+}
+
+.collection-div2 {
+    padding: 1%;
+}
```

Comment:
Created the CSS file and added some classes that are used in the start.html page.

Citation: [https://www.lifewire.com/avoid-inline-styles-for-css-3466846](https://www.lifewire.com/avoid-inline-styles-for-css-3466846)

Reply:
This CSS should provide the exact same look as you had before. I only did an eye-test to prove this, but I had the old page and the updated page open to check, and I noticed no difference.

Reply:
Good.